components/actionbutton/index.css

```diff
@@ -69,7 +29,9 @@
 	--spectrum-actionbutton-border-color-focus: var(--highcontrast-actionbutton-border-color-focus, var(--mod-actionbutton-border-color-focus, var(--spectrum-gray-500)));
 	--spectrum-actionbutton-border-color-disabled: var(--spectrum-disabled-border-color);
 
-	--spectrum-actionbutton-content-color-selected: var(--spectrum-gray-50);
+	--spectrum-actionbutton-content-color-selected: var(--spectrum-gray-25);
+	--spectrum-actionbutton-content-color-disabled: var(--mod-disabled-content-color, var(--spectrum-disabled-content-color));
+	--spectrum-actionbutton-background-color-disabled: var(--mod-disabled-background-color, var(--spectrum-disabled-background-color)); 
```

Comment:
For these 2 new mods (mod-disabeld-content-color and mod-disabled-background-color), what do you think of naming them more in the format we have for the other mods? 

```
--mod-actionbutton-content-color-disabled
--mod-actionbutton-background-color-disabled
```

Just a thought to match the other properties.